data/yam/agama/auto/sle_root_filesystem_ext4.jsonnet

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
you need to remove `sle` from the name of the files as this profiles are generic, no verification needed, I will merge it after that, LGTM.